packages/app/src/app/store/modules/workspace/items.ts

```diff
@@ -1,59 +1,79 @@
 import getTemplate from '@codesandbox/common/lib/templates';
 
-const PROJECT = {
+export interface INavigationItem {
+  id: string;
+  name: string;
+  hasCustomHeader?: boolean;
+  defaultOpen?: boolean;
+  /**
+   * If the item is not applicable in the current situation we sometimes still
+   * want to show it because of visibility. This boolean decides that.
+   */
+  showAsDisabledIfHidden?: boolean;
+}
+
+const PROJECT: INavigationItem = {
   id: 'project',
   name: 'Sandbox Info',
 };
 
-const PROJECT_TEMPLATE = {
+const PROJECT_TEMPLATE: INavigationItem = {
   ...PROJECT,
   name: 'Template Info',
 };
 
-const PROJECT_SUMMARY = {
+const PROJECT_SUMMARY: INavigationItem = {
   id: 'project-summary',
   name: 'Sandbox Info',
   hasCustomHeader: true,
 };
 
-const FILES = {
+const FILES: INavigationItem = {
   id: 'files',
   name: 'Explorer',
   hasCustomHeader: true,
   defaultOpen: true,
 };
 
-const GITHUB = {
+const GITHUB: INavigationItem = {
   id: 'github',
   name: 'GitHub',
+  showAsDisabledIfHidden: true,
 };
 
-const DEPLOYMENT = {
+const DEPLOYMENT: INavigationItem = {
   id: 'deploy',
   name: 'Deployment',
+  showAsDisabledIfHidden: true,
 };
 
-const CONFIGURATION = {
+const CONFIGURATION: INavigationItem = {
   id: 'config',
   name: 'Configuration Files',
 };
 
-const LIVE = {
+const LIVE: INavigationItem = {
   id: 'live',
   name: 'Live',
+  showAsDisabledIfHidden: true,
 };
 
-const MORE = {
-  id: 'more',
-  name: 'More',
-};
-
-const SERVER = {
+const SERVER: INavigationItem = {
   id: 'server',
   name: 'Server Control Panel',
 };
 
-export default function getItems(store) {
+export function getDisabledItems(store: any): INavigationItem[] {
```

Comment:
Maybe create a custom hook instead of passing the store? 🤔

Reply:
This one is used in multiple places, to make it reusable outside of React I think the store given is a nice solution.